sites/gc-page-feedback/includes/strings.liquid

```diff
@@ -0,0 +1,58 @@
+{%- comment -%}
+Set string variables
+{%- endcomment -%}
+
+{%- assign opuft_yes_en = "Yes" -%}
```

Comment:
@EricDunsworth Why a different translation logic for these strings?

Reply:
> @EricDunsworth Why a different translation logic for these strings?
You should probably walk me through your thought process. I see that you use some variables as abstractions. IMHO I would try to separate the logic in a different file and keep the `.liquid` for data storage only.

Reply:
> Why a different translation logic for these strings?
PP wasn't keen on placing feature-specific translations in theme-wide i18n files like ``_data/i18n.yml``, ``sites/includes/i18n.liquid`` or ``sites/includes/i18n-core.liquid``. So I set it up in a feature-specific Liquid file.

> You should probably walk me through your thought process. I see that you use some variables as abstractions. IMHO I would try to separate the logic in a different file and keep the ``.liquid`` for data storage only.
The way I set it up was to use ``sites/gc-page-feedback/includes/strings.liquid`` for all of the plugin's "set-in-stone" strings - like anything that appears on the front-end and checkbox values (derived from their front-end labels).

On the other hand, I used ``sites/gc-page-feedback/includes/parameters.liquid`` for dynamic values. Like anything that's unique to the current page. So for example, institution, page title, page URL, theme, section, etc...